npm/ng-packs/packages/schematics/src/models/method.ts

```diff
@@ -50,12 +51,12 @@ export class Body {
       ? shouldQuote(paramName)
         ? `${descriptorName}['${paramName}']`
         : `${descriptorName}.${paramName}`
-      : nameOnMethod;
+      : camelizeHyphen(nameOnMethod);
```

Comment:
two intended Ternary operators is not well for readability. Maybe we should change "If" statement